system/duplicated_node_checker/config/duplicated_node_checker.param.yaml

```diff
@@ -2,3 +2,5 @@
   ros__parameters:
     update_rate: 10.0
     add_duplicated_node_names_to_msg: true # if true, duplicated node names are added to msg
+    nodes_to_ignore:  # List of nodes to ignore when checking for duplicates
+      - /rviz
```

Comment:
Could you also create the PR for [autoware_launch](https://github.com/tier4/autoware_launch/blob/9b70c1b921d5dadbb8d20280d5348331775eb08a/autoware_launch/config/system/duplicated_node_checker/duplicated_node_checker.param.yaml)?

Reply:
Thank you! I made a PR to OSS launch repo: https://github.com/autowarefoundation/autoware_launch/pull/1070